packages/metrics_center/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   _discoveryapis_commons: ^1.0.0
   crypto: ^3.0.1
   gcloud: ^0.8.2
-  googleapis: ^3.0.0
+  googleapis: '>=3.0.0 <13.0.0'
```

Comment:
Wow, this is Pigeon-level version revving 😁

Reply:
We could honestly do `^12.0.0` I think.

Reply:
Normally I'd say we should favor keeping the broad compatibility, but since no packages depend on `metrics_center` it probably doesn't matter either way.

Reply:
Let's do it then!

Reply:
Bumped to ^12.0.0, see what happens.